3rdparty/perfparser.cmake

```diff
@@ -1,5 +1,6 @@
 include_directories(
-    ${LIBELF_INCLUDE_DIR}
+    ${LIBELF_INCLUDE_DIRS}
+    ${DWARF_INCLUDE_DIR}
```

Comment:
this should be `LIBDWARF_INCLUDE_DIRS`

Reply:
FindElfUtils.cmake uses DWARF_INCLUDE_DIR:

```find_path (DWARF_INCLUDE_DIR
    NAMES
      dwarf.h
    PATHS
      /usr/include
      /usr/local/include
      /opt/local/include
      /sw/include
      ENV CPATH) # PATH and INCLUDE will also work
```

Reply:
Nevermind, later there is a `set (LIBDWARF_INCLUDE_DIRS  ${DWARF_INCLUDE_DIR} ${LIBDW_INCLUDE_DIR})`